scss/_root.scss

```diff
@@ -123,6 +123,11 @@
   // By default, there is no `--bs-focus-ring-x`, `--bs-focus-ring-y`, or `--bs-focus-ring-blur`, but we provide CSS variables with fallbacks to initial `0` values
   --#{$prefix}focus-ring-box-shadow: var(--#{$prefix}focus-ring-x, 0) var(--#{$prefix}focus-ring-y, 0) var(--#{$prefix}focus-ring-blur, 0) var(--#{$prefix}focus-ring-width) var(--#{$prefix}focus-ring-color);
   // scss-docs-end root-focus-variables
+
+  --#{$prefix}form-valid-color: #{$green-500};
```

Comment:
Shouldn't we have new Sass variables like instead of using directly the color Sass variables?
For example:
* `$form-valid-color`
* `$form-valid-border-color`
* `$form-invalid-color`
* `$form-invalid-border-color`

and

* `$form-valid-color-dark`
* `$form-valid-border-color-dark`
* `$form-invalid-color-dark`
* `$form-invalid-border-color-dark`

If we do that, we must think to mention them in the "Sass variables" section of the Form doc page as well.